index.js

```diff
@@ -266,6 +268,21 @@ class bluetoothLE extends EventEmitter {
   static padHex(value) {
     return (`00${value.toString(16).toUpperCase()}`).slice(-2);
   }
+
+  static bytes2hex(bytes, noGaps) {
+    let message = '';
+    for (let i = 0; i < bytes.length; i += 1) {
+      const hex = bytes[i].toString(16).toUpperCase();
+      if (hex.length === 1) {
+        message += '0';
+      }
+      message += hex;
+      if (!noGaps) {
+        message += ' ';
+      }
```

Comment:
Nit: could simplify this by accepting the gap as input string rather than a boolean, defaulting to '' (empty string). Something like `gapStr = gapStr || '';`